python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -60,12 +60,14 @@ def __init__(self, namePattern='', localDir='', **kwds):
             namePattern (str): is the pattern of the output file that has to be written into mass storage
             localDir (str): This is the optional local directory of a file to be uploaded to mass storage
         """
+        self._checkConfig()
```

Comment:
Yes - I am not sure why the current tests work though. The old path should be defunct.